addons/portal/wizard/portal_wizard.py

```diff
@@ -70,116 +76,159 @@ class PortalWizardUser(models.TransientModel):
     email = fields.Char('Email')
-    in_portal = fields.Boolean('In Portal')
-    user_id = fields.Many2one('res.users', string='Login User')
-
-    def get_error_messages(self):
-        emails = []
-        partners_error_empty = self.env['res.partner']
-        partners_error_emails = self.env['res.partner']
-        partners_error_user = self.env['res.partner']
-
-        for wizard_user in self.with_context(active_test=False).filtered(lambda w: w.in_portal and not w.partner_id.user_ids):
-            email = extract_email(wizard_user.email)
-            if not email:
-                partners_error_empty |= wizard_user.partner_id
-            elif email in emails:
-                partners_error_emails |= wizard_user.partner_id
-            user = self.env['res.users'].sudo().with_context(active_test=False).search([('login', '=ilike', email)])
-            if user:
-                partners_error_user |= wizard_user.partner_id
-            emails.append(email)
-
-        error_msg = []
-        if partners_error_empty:
-            error_msg.append("%s\n- %s" % (_("Some contacts don't have a valid email: "),
-                                '\n- '.join(partners_error_empty.mapped('display_name'))))
-        if partners_error_emails:
-            error_msg.append("%s\n- %s" % (_("Several contacts have the same email: "),
-                                '\n- '.join(partners_error_emails.mapped('email'))))
-        if partners_error_user:
-            error_msg.append("%s\n- %s" % (_("Some contacts have the same email as an existing portal user:"),
-                                '\n- '.join([p.email_formatted for p in partners_error_user])))
-        if error_msg:
-            error_msg.append(_("To resolve this error, you can: \n"
-                "- Correct the emails of the relevant contacts\n"
-                "- Grant access only to contacts with unique emails"))
-        return error_msg
-
-    def action_apply(self):
-        self.env['res.partner'].check_access_rights('write')
-        """ From selected partners, add corresponding users to chosen portal group. It either granted
-            existing user, or create new one (and add it to the group).
+
+    user_id = fields.Many2one('res.users', string='User', compute='_compute_user_id')
+    login_date = fields.Datetime(related='user_id.login_date', string='Latest authentication')
+    is_portal = fields.Boolean('Is Portal', compute='_compute_group_details')
+    is_internal = fields.Boolean('Is Internal', compute='_compute_group_details')
+
+    @api.depends('partner_id')
+    def _compute_user_id(self):
+        for portal_wizard_user in self:
+            user = portal_wizard_user.partner_id.with_context(active_test=False).sudo().user_ids
+            portal_wizard_user.user_id = user[0] if user else False
+
+    @api.depends('user_id', 'user_id.groups_id')
+    def _compute_group_details(self):
+        for portal_wizard_user in self:
```

Comment:
I really don't like portal.wizard.user naming. Do you think confident enough to change its naming or do you think "could belong to another task because it is only Tde annoying change requests" ?

Reply:
@tde-banana-odoo No we can change it in this PR :) WHat do you propose ? `portal.access` & `portal.access.partner` ?